<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDynamicWindow</class>
 <widget class="QWidget" name="EditDynamicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: rgb(255, 255, 255);
border-radius:10px;
border:1px solid rgb(100, 169,139);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>55</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>个人动态</string>
      </property>
     </widget>
     <widget class="ElaPlainTextEdit" name="edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>371</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QtMaterialRaisedButton" name="send_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>460</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>发表</string>
      </property>
     </widget>
     <widget class="QWidget" name="head" native="true">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>464</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#head{
background:rgb(0,188,212);
}</string>
      </property>
      <widget class="QToolButton" name="min_btn">
       <property name="geometry">
        <rect>
         <x>379</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover
{
	background-color:rgba(200, 200, 200,85);
}
QToolButton:pressed
{
	background-color:rgba(220,220,220,85);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="close_btn">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>0</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton:hover 
{
	background-color:rgb(255, 0, 0);
}
QToolButton:pressed
{
	background-color:rgb(170, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ElaPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">elaplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
